sub-packages/bionemo-evo2/src/bionemo/evo2/run/train.py

```diff
@@ -492,7 +491,7 @@ def parse_args(args: Optional[List[str]] = None) -> argparse.Namespace:
         help="Disable saving the last checkpoint.",
     )
     parser.add_argument("--lora-finetune", action="store_true", help="Use LoRA fine-tuning", default=False)
-    parser.add_argument("--lora-checkpoint-path", type=Path, default=None, help="LoRA checkpoint path")
+    parser.add_argument("--lora-checkpoint-path", type=str, default=None, help="LoRA checkpoint path")
```

Comment:
in predict.py, lora path is a Path. please keep it consistent. Also, the same comment applies here as above related to the description of the param